Remotion/Web/Development.WebTesting/HostingStrategies/IisExpressProcessWrapper.cs

```diff
@@ -72,7 +71,8 @@ public void Dispose ()
     {
       if (_iisProcess != null && !_iisProcess.HasExited)
       {
-        _iisProcess.CloseMainWindow();
+        _iisProcess.Kill();
+        _iisProcess.WaitForExit();

```

Comment:
What does the documentation say about Kill + WaitForExit?

Reply:
> The Kill method executes asynchronously. After calling the Kill method, call the WaitForExit method to wait for the process to exit, or check the HasExited property to determine if the process has exited. ([docs](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.kill?view=netframework-4.8))